tools/test-tools/bin/assign-test-ports

```diff
@@ -1,3 +1,12 @@
 #!/usr/bin/env node
```

Comment:
I'd really like a `--help` mode or something that tells me about the command line argument. It's not very discoverable unless I also read the part of the README that mentions it can take an argument.

Reply:
Totally agree, I was halfway through making it into standalone mjs scripts when I realized that in order to use the `getTestPort` function, packages would have to import the file by path, and each one would require a potentially different path in each case (depending on how "deep" each example package is, for example).

I also considered making test-tools part of the client release group, to avoid the change-publish-consume dance, but then noticed build-tools also uses it right now. Although now that I think about it, it probably doesn't need to? It has no jest tests... So making this a `@fluid-private` part of the client release group starts sounding like a good option.

The only caveat to all this is that I still need to publish a version with the fix so we can consume it in LTS (or we also merge it into the client release group in LTS).